.drone.star

```diff
@@ -595,15 +596,14 @@ def uploadScanResults(ctx):
                                 "git checkout $DRONE_COMMIT",
                             ],
             },
-        ] + skipIfUnchanged(ctx, "unit-tests") + [
             {
                 "name": "sync-from-cache",
                 "image": MINIO_MC,
                 "environment": MINIO_MC_ENV,
                 "commands": [
                     "mkdir -p cache",
                     "mc alias set cachebucket $MC_HOST $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY",
-                    "mc mirror cachebucket/$CACHE_BUCKET/%s/%s/cache cache/" % (ctx.repo.slug, ctx.build.commit + "-${DRONE_BUILD_NUMBER}"),
+                    "mc mirror cachebucket/$CACHE_BUCKET/%s/%s/cache cache/ || true" % (ctx.repo.slug, ctx.build.commit + "-${DRONE_BUILD_NUMBER}"),
```

Comment:
I removed the `skipIfUnchanged` step and made steps `sync-from-cache` and `purge-cache` not fail even if mc actions failed. This allows `codacy` and `sonarcloud` steps to run even if go-lint pipelines are skipped for some reason. This is not the nicest solution though

CC @micbar @ScharfViktor